go.mod

```diff
@@ -42,6 +41,7 @@ require (
 	github.com/spf13/cast v1.7.1 // indirect
 	github.com/subosito/gotenv v1.6.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
+	golang.org/x/exp v0.0.0-20250103183323-7d7fa50e5329 // indirect
```

Comment:
I'm surprised to see a go.mod change when you are renaming a method

Reply:
I'm sorry. I'll be careful.

Reply:
It's not a matter of being careful. I was simply curious why it changed

And yes, someone looking for commit history about a bug/change cause could be confused